### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by VictoriaStv**
I remember to remove load from favourites. Now, for some reason, when I delete it from index.html and paste it separately into home and exercises, everything breaks.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { renderCards } from './js/categories/categories-api';
 import './api-example';
 import './js/burger-menu';
 import './js/modal/modal';
+import './js/footer/subscription';
```
Comment (hrebynakha) [inline src/main.js:10]: update it to 
```
import './js/Footer/subscription';
import './js/Footer/animation';
```
case sensetive

File: src/js/Footer/subscription.js
Comment (hrebynakha) [inline src/js/Footer/subscription.js:1]: make it as export function

and rewrite with api using
```
import api from '../api';
```
usage:
```
 api.subscription.subscribe(email);
```



add checking form element to function
```
 if (!subscribeForm) {
    return;
  }
```

File: src/js/Footer/animation.js
Comment (hrebynakha) [inline src/js/Footer/animation.js:1]: u can add this like function
```
export function animate() {
  if (!scrollToTopButton) {
    return;
  }
  scrollToTopButton.addEventListener('click', function (event) {
    event.preventDefault();

    window.scrollTo({
      top: 0,
      left: 0,
      behavior: 'smooth',
    });
  });
}
```
after export u can use it in home route after page loaded or wherever u want

**Review by hrebynakha — COMMENTED** [review]
u can load footer in **home.html** and remove it from **index.html**
```
<load src="./partials/footer.html" />
```

after all manipulation u got this on favorites

image

footer on home page will be the same as now

**Review by snegurova — APPROVED** [review]

**Pull request merged**